__init__.py

```diff
@@ -79,24 +83,35 @@ def handle_number_of_languages_intent(self, _):
 
     @intent_handler(IntentBuilder("portal_intent").require("portal_keyword").build())
     def handle_portal_intent(self, _):
-        path, files = self.get_reference_files("/sounds/portal", "mp3")
+        path, files = self.get_reference_files("sounds/portal", "mp3")
         if len(files):
             mp3 = path + "/" + random.choice(files)
-            self.play_audio(mp3)
+            if self.ocp:
+                self._play_in_ocp(mp3, title="Portal Easter Egg")
+            else:
```

Comment:
how do we ever hit this else if ocp is always initialized?

Reply:
Was wondering the same but couldn't work out exactly how we can check if OCP is enabled. In cases of HiveMind we'll have situations where it isn't, right? How can I validate that?

Reply:
hivemind usage here is not supported yet, but once i get to it no changes will be needed, its just a matter of routing messages properly which skills shouldnt worry about

OCP is also supposed to always be available in OVOS, its a direct requirement of ovos-audio  (and soon will be it's own service, but thats besides the point, its still a default component)

for OCP to not be available, its a unsupported/custom OVOS setup by definition

Reply:
I'll remove this use case then